site/verifyemail.html.jet

```diff
@@ -0,0 +1,18 @@
+{{extends "templates/application/application.jet"}}
+
+{{block head()}}
+  {{yield seo(title=i18n("signup_page_header"), index=false)}}
+{{end}}
+
+{{block body()}}
+  <main id="main" class="page form-page">
+    <div class="page-header">
+      <h1>Verify your email</h1>
+    </div>
+    <div class="page-content text-center">
```

Comment:
:x: `text-center`... that's a Bootstrap class... a one line Bootstrap class with `!important` 🤮 

To be consistent with core-template, I would swap that `form-page` class on the main element (this isn't a form page) for `verifyemail-page` and add CSS for this:

```
.verifyemail-page {
  .page-content {
    text-align: center;
  }
}
```

Reply:
Right so we are wanting to never directly use the bootstrap helper classes. I'll update to follow suit (in the interest of time I'm not going to update the other places in core-template that use `text-center`).

I'm guessing the reasoning behind using more singular classes is to support customizations easier? I.e. you only need to override the `.verifyemail-page` class?

Reply:
Yup that's correct.